README.md

```diff
@@ -711,6 +712,40 @@ function findInlineScriptsWithNextData() {
 
 console.log(findInlineScriptsWithNextData());
 
+```
+
+### Inline CSS Info and Size
+
+Find all inline style tags and list them in a table with individual and total byte size. Customize the table below.
+
+```javascript
+
+// Wait for the page to fully load
+
+function findAllInlineCSS() {
+  const convertToKb = (bytes) => bytes / 1000;
+  const inlineCSS = document.querySelectorAll("style");
+  let totalByteSize = 0;
+  for (const css of [...inlineCSS]) {
+    const html = css.innerHTML;
+    const size = new Blob([html]).size;
+    css.byteSize = convertToKb(size) + " kb";
+    totalByteSize += size;
+  }
+  // customize table here, can right click on header in console to sort table
+  console.table(inlineCSS, [
+    "baseURI",
+    "parentElement",
+    "byteSize",
```

Comment:
Add the unit in the header
```suggestion
    "byteSize (kB)",
```

Reply:
I will have to add that to the object itself, you cannot chage the header, i can make it byteSizeInKb

Reply:
or i guess i could have used bracket syntax lol